docs/troubleshooting-remedies/correctly-set-aws-environment-variables.md

```diff
@@ -0,0 +1,27 @@
+# Correctly set AWS Environment Variables
+
+## Problem Summary
+
+If the `chs-dev troubleshoot analyse` command suggests setting your AWS environment variables, check the following:
+1. AWS_PROFILE
+2. _AWS_PROFILE
```

Comment:
This is a Companies House var and has no bearing on `aws` CLI at all - probably irrelevant?

the `_` env vars are used internally by `l_aws` to store bits that are not pertinent to AWS as such so I wouldn't tie `chs-dev` to them